src/handler/http/request/stream/mod.rs

```diff
@@ -841,57 +826,68 @@ async fn get_super_cluster_delete_status(
     // For each node in the super cluster, get the delete status
     let trace_id = config::ider::generate_trace_id();
     let mut results = Vec::new();
-    let mut all_pending = false;
+    let mut any_pending = false;
     let mut all_completed = true;
     let mut errors = Vec::new();
 
     for cluster in clusters {
         match crate::service::cluster_info::get_super_cluster_delete_job_status(
             &trace_id,
             cluster.clone(),
-            org_id,
-            stream_type,
-            stream_name,
-            time_range,
+            id,
         )
         .await
         {
             Ok(response) => {
-                if !response.is_complete {
-                    all_pending = true;
+                let job_status = CompactorManualJobStatus::from(response.status);
+                if job_status == CompactorManualJobStatus::Pending {
+                    any_pending = true;
+                } else if job_status != CompactorManualJobStatus::Completed {
```

Comment:
I think both are same condition if I am not missing anything. Today, we only consider pending and completed, if job_status! == completed, it is pending, which is the first if statement. If the status is pending, it will never go to the second one. I think we can put the all_completed = false; inside the first if statement itself.